app/main.py

```diff
@@ -1 +1,42 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            count_of_eaten = self.appetite
+            self.is_hungry = False
+            return count_of_eaten
+        else:
+            return 0
```

Comment:
```suggestion
        return 0
```
Тут else вже не потрібно, ми можемо відразу вказати `return 0` для решти випадків.